app/controllers/organiser_registrations_controller.rb

```diff
@@ -0,0 +1,37 @@
+class OrganiserRegistrationsController < ApplicationController
+ before_action :require_organiser
```

Comment:
when you click on this link from your inbox you won't be logged in to the system so we can't require that user will be logged it (he can't do that yet - that's why we're showing this page ;)) so please remove this `before_filter`